Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -15,11 +15,12 @@ <h3>Trapping<h3>
 <p>
     If a player traps a citizen of Sudan, they will be required to give the victim their stuff
     back. If they have a high ranking position such as military member or minister in the nation it may be 
-    removed.
+    removed. A fine of up to 10g may be issued.
 <p>
 <h3>Fines</h3>
 <p>
-    If a citizen doesn't pay a fine, they won't be able to run for a Ministirial position or join the Military until
+    If a citizen doesn't pay a fine, they won't be able to run for a Ministerial position or join the Military until
     that fine is payed off. If a fine is not payed back after 2 weeks, the amount fined will go up by 1G each
-    week until it's payed back.
+    week until it's payed back. If a member of the Military or a person who is in a Ministerial position has a fine
+    that they havent paid back after 2 weeks they will be removed from their position.
```

Comment:
There are a few grammatical errors.
```suggestion
    week until it's paid back. If a member of the Military or a Minister has a fine
    that they haven't paid back after 2 weeks, they will be removed from their position.
```